### PR TITLE
Broadcast playlist when new guest joins

### PR DESCRIPTION
**Pull request by bryansills**

### CODE REVIEW

File: SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java [hunk 1]
```diff
@@ -153,8 +153,14 @@ public void onReceiveAction(Context context, Intent intent) {
                 startDataManager();
             }
         })
+        .addAction(ConnectionService.ACTION_GUEST_CONNECTED, new IBroadcastActionHandler() {
+            @Override
+            public void onReceiveAction(Context context, Intent intent) {
+                ((CustomApp)getApplication()).getMessagingService().sendPlaylistMessage(mPlaylist.getSongsToPlay());
```
Comment (reidbaker) [inline SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java:159]: This line is pretty long

**Comment by reidbaker** [comment]
Other than the long line I think this looks pretty good. Fix that and :shipit:

File: SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java (hunk 1 above)
Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/service/PlaylistService.java:159]: This line is going to get changed when I submit a pull request to fix the MusicLibraryService, so I don't think this is a show stopper for now.

**Comment by theJenix** [comment]
Looks good to me, with note about the long line, :thumbsup:

**Pull request merged**